pkg/common/config/types.go

```diff
@@ -51,6 +51,9 @@ type Config struct {
 		VCClientTimeout int `gcfg:"vc-client-timeout"`
 		// Cluster Distribution Name
 		ClusterDistribution string `gcfg:"cluster-distribution"`
+
+		//CSIAuthCheckIntervalInMin specifies the interval that the auth check for datastores will be trigger
+        CSIAuthCheckIntervalInMin int `gcfg:"csi-auth-check-intervalinmin"`
```

Comment:
@lipingxue 
Can you add an entry for `csi-auth-check-intervalinmin` in this file : example/vanilla-k8s-block-driver/vsphere.conf

Reply:
Added.